services/QuillLMS/app/services/gengo/request_translations.rb

```diff
@@ -24,7 +26,9 @@ def run
     end
 
     def gengo_payload
-      english_texts.reduce({}) do |hash, english_text|
```

Comment:
`each_with_object` is a nice ruby pattern for reduce/merge; also, begin/end block will help with getting formatting with memoized variable:

```
   @gengo_payload ||= begin
      english_texts
        .select(&:needs_translation?)
        .each_with_object({}) do |english_text, hash|
          hash[english_text.id.to_s] = {
            type: "text",
            body_src: english_text.text,
            lc_src: "en",
            lc_tgt: Gengo::SPANISH_LOCALE,
            tier: "standard",
            slug: english_text.id,
            group: true,
            auto_approve: true,
            comment: STANDARD_COMMENT
          }
      end
    end
```